### ISSUE TITLE
Support for TanStack Start

### ISSUE THREAD

**Issue by thomasballinger** [get-convex/convex-auth#126]
People in the Convex Discord have been trying this lately, I think it's going to require adding some code in this library to do it nicely, similar to Next.js middleware. Filing for tracking, ETA is in the next two weeks.

**Comment by Mark-Pineda-C** [comment]
any updates on this?

**Comment by igor9silva** [comment]
Yes, we don't have cookie-support yet AFAIK, making SSR auth not viable

**Comment by IgnisDa** [comment]
Hi! Has there been any progress on this?
